corehq/apps/app_manager/models.py

```diff
@@ -986,9 +986,91 @@ def validate(self, value, required=True):
 
 
 class NavMenuItemMediaMixin(DocumentSchema):
+    """
```

Comment:
This was the initial direction I took, I used 'en' by default. But I went with this solution as this gives more unambiguous. For example an app that doesn't have 'en' as one of the languages will have 'default' key in this approach instead of 'en'. Apart from that, all of the 'default' logic is automatically taken care of in the interfacing methods.